numba/unicode.py

```diff
@@ -622,6 +622,29 @@ def join_str_impl(sep, parts):
         return join_str_impl
 
 
+@overload_method(types.UnicodeType, 'zfill')
+def unicode_zfill(str_to_fill, width_to_fill):
```

Comment:
Please fix the names of arguments so that `unicode_zfill` and `zfill_impl` have the same argument names.